### DISCUSSION TITLE
Where is the "clipping planes"? I can't find it!

### DISCUSSION THREAD

**Discussion by kstcmcjw** [nrc-cnrc/EGSnrc#843] [Questions] [answered]
I have studied the chapter 6, about the concentric spheres, and i edit the program:

:start geometry definition:
     :start geometry:
      name = my_spheres # this name is up to you
      library = egs_spheres
      radii = 0.3 1.8 2 # three concentric spheres
        :start media input:
           media = default shell center # names of media 0 1 2 ...
           set medium = 2 1 # set region 2 to medium 1 (shell)
           set medium = 0 2 # set region 0 to medium 2 (center)
        :stop media input:
    :stop geometry:
simulation geometry=my_spheres
:stop geometry definition:

however, i achieve the simulation of the concentric spheres, as the figure show:
![image](https://user-images.githubusercontent.com/96934217/151928441-2941f860-6398-4660-882d-045eaa9862cf.png)
![image](https://user-images.githubusercontent.com/96934217/151928536-dca844e9-57f5-4f7e-ae43-a8d4451347de.png)

there still a question: this simulation show the clipping planes as this figure:

![image](https://user-images.githubusercontent.com/96934217/151928657-81b9a4de-b13d-4c18-bc6c-f3e5f7401d3a.png)

and the EGSnrc getting-strat.pdf show that clipping way:

![image](https://user-images.githubusercontent.com/96934217/151928888-60bd3a7c-7f5b-45ee-96f0-c1904d470e25.png)

my question is how to achieve the clipping planes? I can't find any useful information on the Report PIRS-898 (2019), the website is https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/index.html
there is no information of "clipping planes"?

Thanks for your answer！

**Comment by rtownson** [comment, marked as answer]
In `egs_view` look in the `Display` tab instead of the `Colors`. You will see a section on the left that looks like a table. In each row here, you can enter a unit vector for the normal to the plane, and a distance from the origin. E.g. try `1 0 0 0`. The 4th number is the distance.

**Comment by kstcmcjw** [comment]
> 在选项卡`egs_view`中查看`Display`而不是`Colors`. 您将在左侧看到一个看起来像表格的部分。在此处的每一行中，您可以输入平面法线的单位矢量以及距原点的距离。例如尝试`1 0 0 0`。第四个数字是距离。

Thank you very much, i get it !!
![image](https://user-images.githubusercontent.com/96934217/152371235-96593d1f-ae8b-44ee-a9ab-f5f4f65257b0.png)
